node_modules/app/http.js

```diff
@@ -2,16 +2,22 @@
 
 var Task = require('data.task')
 var xhr = require('xhr')
-var { curry, prop, compose, map } = require('ramda')
+var { curry, prop, compose, map, identity } = require('ramda')
 
-var get = (path) => request('get', path, {})
-
-var post = curry((body, path) => request('post', path, { json: body }))
+var basic_request = curry((method, body, path) =>
```

Comment:
Let's not do this, as it leaves us with multiple APIs for async actions, which is confusing